### PR TITLE
fix axisym source term + update regression cases

### PR DESCRIPTION
**Pull request by FlorianDm**
## Proposed Changes

fix https://github.com/su2code/SU2/issues/1365

### CODE REVIEW

**Comment by FlorianDm** [comment]
I updated the restart file in the TestCases directory so I suppose regressions apart from this branch will break until this PR is merged. Sorry

**Comment by pcarruscag** [comment]
I have some vague memory of the textbook having a typo?

**Comment by WallyMaier** [comment]
@pcarruscag I also have a vague memory of that.  But I couldn't seem to find it in the previous discussions....

**Comment by FlorianDm** [comment]
Yes it did but it's unrelated. Surprised this bug here could creep in. For the regression it seems like it is still using the old restart file and not the one I pushed to TestCases master a few hours ago. Not sure what I am missing here

**Comment by FlorianDm** [comment]
@WallyMaier OK will do

**Comment by WallyMaier** [comment]
@FlorianDm, not sure the issue either.  You could just do a "cold" start for the regression?

**Comment by pcarruscag** [comment]
Don't push things to master directly
create a PR to develop and then merge, so we can keep track of things.
It's develop you need not master

**Comment by FlorianDm** [comment]
Yes just realised that sorry
Will fix it

File: SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp [hunk 1]
```diff
@@ -378,7 +378,7 @@ CNumerics::ResidualType<> CSource_NEMO::ComputeAxisymmetric(const CConfig *confi
     residual[nSpecies+1] -= Volume*(yinv*total_viscosity_i*2*(GV[nSpecies+3][1]-v*yinv)
                                                              -TWO3*AuxVar_Grad_i[0][1]);
     residual[nSpecies+2] -= Volume*(yinv*(-sumJhs_y + total_viscosity_i*(u*(GV[nSpecies+3][0]+GV[nSpecies+2][1])
-                                                                        +v*TWO3*(2*GV[nSpecies+2][1]-GV[nSpecies+2][0]
+                                                                        +v*TWO3*(2*GV[nSpecies+3][1]-GV[nSpecies+2][0]
```
Comment (WallyMaier) [inline SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp:381]: Thanks!

**Review by WallyMaier — APPROVED** [review]
This looks good to me! Im still surprised how this slipped through.

**Pull request merged**